code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -195,6 +196,9 @@
 		owner.update_hair()
 		owner.update_damage_overlays()
 
+obj/item/bodypart/proc/set_fakeness(fakeness)
```

Comment:
Setter doesn't seem necessary here. Or at least i can't think of a scenario where it should have side effects.